pdns/dnsdist.cc

```diff
@@ -1199,6 +1202,7 @@ try
   }
   
 #endif
+  ComboAddress clientAddress = ComboAddress();
```

Comment:
Sure, I had something like [1] in mind, but I'm ok either way.

[1]: https://github.com/pieterlexis/pdns/compare/dnsdist-client-improvements...rgacogne:dnsdist-client-improvements?expand=1